types/signatures.go

```diff
@@ -57,7 +57,7 @@ type (
 		SiafundOutputs           []uint64
 		MinerFees                []uint64
 		ArbitraryData            []uint64
-		Signatures               []uint64
+		TransactionSignatures    []uint64
```

Comment:
The problem is that there's a TransactionSignature type and a Signature type, and it's weird to call Trasaction.Signatures and not have it reference the Signature type.

I've been waffling over this one for a bit but it has confused me a few times.